content/en/dashboards/functions/rate.md

```diff
@@ -3,6 +3,10 @@ title: Rate
 kind: documentation
 aliases:
     - /graphing/functions/rate/
+further_reading:
+- link: "/monitors/guide/alert-on-no-change-in-value/"
+  tag: "Guide"
```

Comment:
```suggestion
  tag: "Documentation"
```

Reply:
The kind for this page is listed as "guide", do we default to documentation for tags?

Reply:
Yep, we don't explicitly state "Guide" or "FAQ", just "Documentation" usually!